ctapipe/image/hillas.py

```diff
@@ -165,6 +165,39 @@ def hillas_parameters(geom, image):
 
         m4_long = np.average(longitudinal ** 4, weights=image)
         kurtosis_long = m4_long / length ** 4
+    
+    """
+    Compute of the Hillas parameters uncertainties. 
+    Implementation described in [hillas_uncertainties]_ This is an internal MAGIC document
+    not generally accessible.  
+    """
+   
+    #intermediate variables
+    cos_psi = np.cos(2 * psi)
```

Comment:
perhaps call this variable cos_2psi, since it's not cos(psi), just to avoid possible confusion in the future if somebody re-uses it later.

Reply:
Variable's name changed